src/backend/api/meals.js

```diff
@@ -2,14 +2,72 @@ const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
+//select all
 router.get("/", async (request, response) => {
   try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
+    const meals = await knex("meal").select("*");
+    response.json(meals);
+  } catch (error) {
+    throw error;
+  }
+});
+//new meal
+router.post("/", async (request, response) => {
+  const addmeal = request.body;
+  const [mealsId] = await knex("meal").insert({
+    title: addmeal.title,
+    description: addmeal.description,
+    location: addmeal.location,
+    when: addmeal.when,
+    max_reservations: addmeal.max_reservations,
+    price: addmeal.price,
+    created_date: addmeal.created_date,
+  });
+  if (mealsId === 0) {
+    response.status(404).json({ message: "No meals found" });
+  } else {
+    response.status(201).json({ message: "New created meal", id: mealsId });
+  }
+});
```

Comment:
I did a similar message for error but I think the word "found" is more appropriate for router.get method.
maybe No meals created/ added.